x-pack/plugins/security_solution/public/management/pages/blocklist/view/components/blocklist_form.tsx

```diff
@@ -378,7 +380,10 @@ export const BlockListForm = memo(
 
         const nextItem = { ...item, tags };
 
-        setSelectedPolicies(change.selected);
+        // Preserve old selected policies when switching to global
+        if (!change.isGlobal) {
+          setSelectedPolicies(change.selected);
+        }
```

Comment:
Is it east enough to add a test for this bug? 🙏

Reply:
There is no test file currently for blocklist form, I think @joeypoon was working on it, is it right?

Reply:
I see. Should add the test IMHO. Later if not in this PR.